tcf_website/static/base/base.css

```diff
@@ -79,15 +64,16 @@ footer > .row {
 .btn.btn-primary:hover,
 .btn.btn-outline-primary:hover {
   color: white;
-  background-color: #d1420d;
-  border-color: #d1420d;
+  background-color: #AC451E;
```

Comment:
These styles look good although the `Submit Review` button is still default blue when clicked

Reply:
oh yeah bwahah I thought it looked cool
Fine I can change it to our indigo or a darker orange

Reply:
Yeah indigo is fine if you want but as long as we use our brand colors

Reply:
done, kept it at dark orange